src/controller/AutoCommissioner.h

```diff
@@ -23,6 +23,9 @@
 namespace chip {
 namespace Controller {
 
+constexpr size_t kAttestationElementsLength  = 900;
+constexpr size_t kAttestationSignatureLength = 64;
```

Comment:
This is `chip::Crypto::kMax_ECDSA_Signature_Length` from CHIPCryptoPAL.h, please use that instead

Reply:
addressed